src/hooks/useSelectedTransactionsActions.ts

```diff
@@ -33,6 +41,16 @@ function useSelectedTransactionsActions({
     const {selectedTransactionsID, setSelectedTransactionsID} = useMoneyRequestReportContext();
     const {translate} = useLocalize();
     const [isDeleteModalVisible, setIsDeleteModalVisible] = useState(false);
+    const isTrackExpense = isTrackExpenseReport(report);
```

Comment:
I think this describes it better since this is a report, not a transaction
```suggestion
    const isTrackExpenseThread = isTrackExpenseReport(report);
```